web_graph_sort/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Web Graph Sort
+==============
+
+This module allows to sort pivot tables.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to ...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/web/8.0
```

Comment:
This link is not correct. Please see README template (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)